docs/CHANGELOG.rst

```diff
@@ -59,7 +60,7 @@ Changed
   ``normalization_type`` input.
 - Bump MultiQC to version 1.7.0 in ``multiqc`` process
 - Use ``resolwebio/common:1.1.0`` base image for the
-  ``resolwebio/rnaseq:4.3.0`` Docker image
+  ``resolwebio/rnaseq:4.4.0`` Docker image
```

Comment:
Sorry to bug you and not being more clear before, but this line should be omitted. 
The introduction of ``resolwebio/common:1.1.0`` to ``resolwebio/rnaseq:4.3.0`` happened with previous release, and your commit does't change that.